src/layers-gov.json

```diff
@@ -26,7 +27,8 @@
       "python3.9": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Python39:52",
       "python3.9-arm": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Python39-ARM:52",
       "extension": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Extension:21",
-      "extension-arm": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Extension-ARM:21"
+      "extension-arm": "arn:aws-us-gov:lambda:us-gov-east-1:002406178527:layer:Datadog-Extension-ARM:21",
+      "dotnetcore3.1": "arn:aws:lambda:us-gov-east-1:464622532012:layer:dd-trace-dotnet:1"
```

Comment:
Looks like the GovCloud layers still have `dotnetcore3.1` instead of `dotnet` here.